apps/framework-cli/src/cli/routines/metrics_console/run_console.rs

```diff
@@ -24,18 +25,35 @@ pub async fn run_console() -> app::AppResult<()> {
     let mut tui = tui::Tui::new(terminal, events);
     tui.init()?;
 
+    let (tx, mut rx) = tokio::sync::mpsc::channel::<(f64, f64, Vec<(f64, f64, String)>)>(10);
+
+    tokio::spawn(async move {
+        loop {
+            let (average, total_requests, summary) = client::client().await.unwrap();
```

Comment:
Can we use a verb as the function name?

Reply:
Do you want me to wrap this into a function or rename client()?